packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/LineGraph/BasicLineGraphs/Y2Axis.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import lineTimesries from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Line/lineTimeseries';
+import exampleData from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Line/y2AxisData';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '@cerner/terra-graphs/lib/components/Graph.module.scss';
+import '@cerner/terra-graphs/lib/components/Line/LineGraph.module.scss';
+
+/*
+Please refer to the documentation below to see the graphConfig and data objects
+*/
+const graphConfig = utils.deepClone(lineTimesries('#lineY2Axis'));
+graphConfig.axis.y2.show = true;
+const dataset = utils.deepClone(exampleData);
+
+const LineExample = () => {
+  React.useEffect(() => {
+    const graph = Carbon.api.graph(graphConfig);
+    dataset.forEach((data) => {
+      graph.loadContent(Carbon.api.line(data));
+    });
+  }, []);
+  return (
+    <React.Fragment>
+      <div id="tooltip" className="initial-tooltip" />
+      <div id="lineY2Axis" />
+    </React.Fragment>
+  );
+};
+
+export default LineExample;
```

Comment:
Not a big deal but typically the export module name would match the file name (in this case Y2Axis) unless there are multiple exports. I'm seeing that we use the same `LineExample` for all of these examples. This is probably not an issue but just a bit inconsistent with naming we have been using.

Reply:
Fixed in [this](https://github.com/cerner/terra-graphs/pull/180/commits/54b52861307d9ef55bc7df64ff395982154d707a) commit.